### PR TITLE
make spec file compliant again with SUSE (and OBS)

### PR DESCRIPTION
**Pull request by pseiler**
Some minor spec file changes to let it build with SUSE OBS again

### CODE REVIEW

**Comment by jirihnidek** [comment]
@pseiler It looks ok. Can you please squash commits into one and add some comment to that?

**Comment by ptoscano** [comment]
As @jirihnidek  mentioned, please squash the commits, or at least simplify them a bit (possibly by type of change).
Also, please describe in the commit messages what each change is about: we rarely test SUSE, so knowing why a certain bit is needed is essential for us, otherwise we easily risk breaking things again accidentally.

**Comment by pseiler** [comment]
@jirihnidek @ptoscano i squashed them and added a better commit log.

File: subscription-manager.spec [hunk 1]
```diff
@@ -875,6 +880,9 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 # syspurpose
 %dir %{python_sitearch}/syspurpose
 %{python_sitearch}/syspurpose/*.py
+%if 0%{?suse_version}
+%{python_sitearch}/syspurpose/*.pyc
+%endif
```
Comment (ptoscano) [inline subscription-manager.spec:885]: Most likely we can avoid this SUSE-only block by turning the `*.py` pattern in the line before to `*.py*`, in a similar way to what is done few lines above for rhsmlib.

File: subscription-manager.spec [hunk 2]
```diff
@@ -885,6 +893,10 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 
 # Despite the name similarity dbus-1/system.d has nothing to do with systemd
 %config(noreplace) %{_sysconfdir}/dbus-1/system.d/com.redhat.*.conf
+%if 0%{?suse_version}
+%dir %{_sysconfdir}/dbus-1
+%dir %{_sysconfdir}/dbus-1/system.d
```
Comment (ptoscano) [inline subscription-manager.spec:898]: OK, I see why they are needed. One small request: can you please move these two `%dir` together with the other `%dir` for SUSE? See almost 200 lines above:
https://github.com/candlepin/subscription-manager/blob/e16ba11bea4cfcaa80478c907013f9c7d9ef9c28/subscription-manager.spec#L706-L710

**Review by ptoscano — CHANGES_REQUESTED** [review]
Thanks @pseiler for the update, also with the useful commit message!

It is mostly OK from my POV; I noted a couple of things that IMHO could simplify the additions a bit, and reduce the added SUSE-blocks.

File: subscription-manager.spec (hunk 2 above)
Comment (pseiler) [reply subscription-manager.spec:898]: I will work on it. Thank you for the feedback.

**Pull request closed without merging**